bebraven/platform
# Show previous project submissions, allow empty responses

**Task**: https://app.asana.com/0/1174274412967132/1187445581799819, https://app.asana.com/0/1174274412967132/1197654110183105

### Description

This PR implements a bit more of the project submission functionality:
- Render previous project submissions in the Canvas SpeedGrader with the correct responses from the Fellow
- Render empty responses correctly on submissions

**Previous submissions**
- Use the `ProjectSubmission:created_at` timestamp in the TinCan `until` parameter (https://xapi.com/reporting/) when querying for project submission responses
- `until` is the Fellow's next submission's (if it exists) `create_at`. 

**Empty submissions**
- Allow sending empty text to the LRS
- When loading previous responses,  use the most recent statement (with `until` cut off, see above) and mark the input as set (rather than checking for empty)

See https://app.asana.com/0/1174274412967132/1197477442813577 for how we plan on (eventually, maybe) stop using Learning Locker's LRS from project submissions entirely in the future. 

### Testing

- `rspec`
- Login as Fellow and clear out response to project, refresh page, see that the input remains blank after refresh
<img width="1792" alt="Screen Shot 2020-10-07 at 12 33 58 PM" src="https://user-images.githubusercontent.com/62911934/95378927-820f6e80-0899-11eb-9767-b3283b4baa69.png">

- Login as Fellow and submit assignment twice (first response: **123**, second: **abc**)
- View submissions as TA in Canvas SpeedGrader, most recent is **abc**
<img width="1792" alt="Screen Shot 2020-10-07 at 12 30 49 PM" src="https://user-images.githubusercontent.com/62911934/95378656-14fbd900-0899-11eb-8088-6e29d1f11481.png">

- Previous submission is **123**
<img width="1792" alt="Screen Shot 2020-10-07 at 12 30 56 PM" src="https://user-images.githubusercontent.com/62911934/95378658-162d0600-0899-11eb-9838-08f9f8eba7a0.png">

- As Fellow, make more changes to submission, but don't submit
- As TA, verify the most recent submission values don't change

### Types of changes




### Checklist:
